### DISCUSSION TITLE
Adding PANNING interaction disables touch screen poster dismiss.

### DISCUSSION THREAD

**Discussion by jacktasak** [google/model-viewer#3174] [Q&A]
Hi everyone,
When I add the script that allows panning on the model (the one shared here: https://modelviewer.dev/examples/stagingandcameras/#panning), it disables poster dismissing on touch devices.
I'm not familiar with scripting, but as I can imagine this probably is some small isue that may be easily resolved, but I cannot find any help over internet, so I decided to ask for help here.
Thanx in advance!

**Comment by samaneh-kazemi** [comment]
Hi @jacktasak, can you please share a link to your project so we can try it? You can use [glitch](https://glitch.com/).

**Reply by jacktasak** [reply]
Sure, you can check it at https://360creative.pl/model-test/index.html
If you enter https://360creative.pl/model-test/index2.html you can see that it works, but with no panning. Only difference is, that for this "2" version I removed query.selector name from the second line of the panning script.
Both versions work as expected from my windows pc.

**Reply by samaneh-kazemi** [reply]
Thanks for sharing. I'm not able to access it.
![Screen Shot 2022-02-02 at 10 37 01 AM](https://user-images.githubusercontent.com/3915613/152217299-40d0a678-b972-464a-967b-9aa605e7bc74.png)

**Reply by jacktasak** [reply]
I'm really sorry, I messed up the link. It's alreade been corrected (in the post above).

**Reply by elalish** [reply]
Interesting; I'd guess it might be related to one of the `stopPropagation()` calls. Can you do a little JS debug to see what's going on?

**Reply by jacktasak** [reply]
could you guide me thru this? I assume that you don't mean looking into chrome console?

**Reply by elalish** [reply]
yeah, the chrome console and setting some break points. If you're not comfortable with that level of engineering, you might be better off just calling `dismissPoster()` after a timeout instead of setting it for user-interaction.

**Reply by jacktasak** [reply]
Well, I was hoping someone could try to find the problem, because I'm not able to do it by myself. I will probably have to use some workaround, because dismissing poster after fixed time is not an option to consider for this project.

**Comment by jacktasak** [comment]
Today I found out, that adding this panning-script blocks any interactions with model, i.e. click on annotation with some actions added does not work... It would be nice, if someone from model-viewer team could look closer, because it seems that you guys provide bugged example to all users :(

**Reply by elalish** [reply]
well, examples are just that; they are for people to modify and use for their own purposes. It's never a good idea to blindly copy JS. However, we do plan to pull panning support into MV so you can just set an attribute, at which point testing like this will happen.
